Exercises/numbers/numbers.js

```diff
@@ -0,0 +1,50 @@
+//1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+//const decimal = new Number('127');
+
+const binary = '0000000001111111';
+const octal = '0x7F';
+const hexadecimal = '7F'; 
+
+console.log(binary,octal,hexadecimal);
+
+//2. Implement function `extractNumbers` which will leave only numbers in passed string (in the beginning of the string).
+
+function extractNumbers () {
+  price = '999usd';
+  number = parseInt(price); 
+  console.log(number);
+}
+extractNumbers();
+
+//3. Declare a variable with decimal value and convert to binary, octal and hexadecimal.
+
+const numberDecimal = new Number('9');
+
+console.log((numberDecimal).toString(2)); //1001
+console.log((numberDecimal).toString(8)); //011
+console.log((numberDecimal).toString(16)); //9
+
+//4. Use `Math.random` to generate random value from `0` to `100`.
+
+let randomInteger = Math.floor(Math.random() * 101);
+console.log(randomInteger);
+
+//5. Make function which will take a array with numbers as a parameter and calculate average number and round to hundredths.
+
+function arrayCalculation () {
+  let values = [101,202,303,404,505];
+  let sum = values.reduce((a,b) => a + b,0);
+  let average = sum / values.length; 
+  let total =(Math.round(100*average)/100);
+  console.log(total);
+}
+arrayCalculation();
+
+//6. Generate array to pass to function above with values, and length generated by Math.random.
+
+let arrayValues = new Array(Math.floor(Math.random() * 100));
+console.log(arrayValues.length);
+for(var i = 0; i < arrayValues.length; i++) {
+  arrayValues.push(Math.floor(Math.random() * 100));
+}
+arrayCalculation(arrayValues);
```

Comment:
Your `arrayCalculation` function is not expecting any parameters.

So when you call it with `arrayValues` parameter this parameter will be ignored.

To fix that you need to change the function declaration.

For example:

```js
// Function with parameters
function sum(a, b) {
  return a + b;
}

sum(3, 4); // 7
```

Reply:
updated 
function arrayCalculation(values) {
  const sum = values.reduce((a, b) => a + b, 0);
  const average = sum / values.length;
  const total = Math.round(100 * average) / 100;
  console.log(total);
}
const values = [101, 202, 303, 404, 505];
arrayCalculation(values);